code/datums/station_traits/neutral_traits.dm

```diff
@@ -64,3 +64,15 @@
 /datum/station_trait/announcement_baystation/New()
 	. = ..()
 	SSstation.announcer = /datum/centcom_announcer/baystation
+
+/datum/station_trait/random_dept_signs
+	name = "Department Sign Mixup"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 10
+	show_in_report = TRUE
+	report_message = "Our engineers seem to have messed up sign placement. Please bear with us until we can get a repair crew out."  // nobody is coming
+
+/datum/station_trait/random_dept_signs/New()
+	. = ..()
+	for(var/obj/structure/sign/departments/the_sign in GLOB.dept_signs)
```

Comment:
This needs to have a if with station z level check.

Reply:
how do i check that

Reply:
Try to search any zone-related restriction codes, and check how those work.